gui/default/syncthing/core/syncthingController.js

```diff
@@ -2096,14 +2096,15 @@ angular.module('syncthing.core')
             }
         };
 
-        $scope.otherDevices = function (devices) {
-            if (devices === undefined) {
-                devices = $scope.deviceList();
-            }
-            return devices.filter(function (n) {
-                return n.deviceID !== $scope.myID;
-            });
-        };
+    	$scope.otherDevices = function(devices) {
+        	if (devices === undefined) {
+            	devices = $scope.deviceList();
+        	}
+        	const ids = devices.map((device) => device.deviceID).filter((id) => id !== $scope.myID)
```

Comment:
I would suggest to replace `const` with `var` and use regular functions instead of arrow functions. This is because the current code is rather old-school and doesn't use any of those.